src/card.css

```diff
@@ -0,0 +1,106 @@
+
+.card {
+  width: 200px;
+  background-color: var(--backgroundColor);
+  box-sizing: border-box;
+  border: 1px solid var(--borderColor);
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 159px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__img--imac {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card__name {
+  width: 166px;
+  margin-top: 40px;
+}
+
+.card__description {
+  width: 100%;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--darcBlue);
+}
+
+.card__code {
+  height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-top: 4px;
+  color: var(--grey);
+}
+
+.card__conteiner {
+  box-sizing: content-box;
+  padding: 0;
+  margin: 16px auto 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px auto 0;
+}
+
+.card__priceText {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--grey);
+}
+
+.card__value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--darcBlue);
+}
+
+.card__buy {
+  height: 40px;
+  width: 166px;
+  box-sizing: border-box;
+  margin: 16px auto 0;
+  background: var(--blue);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: all 0.5s;
+}
+
+.card__button {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: var(--backgroundColor);
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: all 0.5s;
```

Comment:
Don't set all for transition, it affects on website performance in bad way. Specify only needed properties